src/Mynt.Backtester/DataRefresher.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Mynt.Backtester
 {
-    internal class DataRefresher
+    public class DataRefresher
```

Comment:
When refreshing data I get the following error in my console log:

> Refreshing...
> One or more errors occurred. (Your platform does not support FileStream.Lock. Please set mode=Exclusive in your connnection string to avoid this error.)

Reply:
This seems like an LiteDB error on MacOS -> https://github.com/mbdavid/LiteDB/issues/787

I'll include a platform check and set to exclusive mode where this option is needed.

Reply:
I've rewrite some classes - Please try if it's now working on OSX - I've tested successfully on Windows and Linux